### DISCUSSION TITLE
Export / Import All Server Settings and document Server Settings Location

### DISCUSSION THREAD

**Discussion by benjamin74** [spatie/ray#614] [Ideas]
Hello,

It would be nice to have an **Export / Import All Servers** functionality for backup / migration to another Mac. I have quite a ton and would really hate having to type them back again.

Also could you please document where the Servers settings are located so we can back them up to dropbox or somewhere else through some scripts / automation ?

Thanks,

**Comment by AdrianMrn** [comment]
You can find your preferences and servers in `~/Library/Application Support/Ray/preferences.json`, you can safely overwrite this file with an earlier stored version. I'm not entirely sure, but you can try to copy `ray-encrypted.json` from the same folder as well. With some luck, this will allow Ray to retrieve your server passwords too, if your Keychain has synced.

Could you please let me know if this works for you, and I'll add it to the docs!
